engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -1203,17 +1209,33 @@ private AllocateResponse sendContainerAskToRM(List<ContainerRequest> containerRe
       amRmClient.releaseAssignedContainer(containerId);
     }
 
+    return amRmClient.allocate(0);
+  }
+
+  private void processContainerStopRequests()
```

Comment:
How about calling it processPendingContainers, see comment below on the method format.